src/modules/crafting.haml

```diff
@@ -0,0 +1,354 @@
+                                        %code= html_escape('<crafting> </crafting>')
+                                %td
+                                    Node containing the custom crafting recipes.
+                                %td
+                            %tr
+                                %th{:colspan => "2"} Sub-elements
+                                %th Value/Children
+                            %tr
+                                %td
+                                    %pre
+                                        %code= html_escape('<shaped>')
+                                %td
+                                    A shaped crafting recipe node.
+                                    %a.left-ref-link{:href=>"#shaped"} <i class="fa fa-chevron-down"></i>
+                                %td
+                                    %span.label.label-default Shaped Recipe Sub-elements
+                            %tr
+                                %td
+                                    %pre
+                                        %code= html_escape('<shapeless>')
+                                %td
+                                    A custom shapeless recipe node.
+                                    %a.left-ref-link{:href=>"#shapeless"} <i class="fa fa-chevron-down"></i>
+                                %td
+                                    %span.label.label-default Shapeless Recipe Sub-elements
+                            %tr
+                                %td
+                                    %pre
+                                        %code= html_escape('<smelt>')
+                                %td
+                                    A custom smelting recipe.
+                                    %a.left-ref-link{:href=>"#smelt"} <i class="fa fa-chevron-down"></i>
+                                %td
+                                    %span.label.label-default Smelt Recipe Sub-elements
+                            %tr
+                                %td
+                                    %pre
+                                        %code= html_escape('<disable>')
+                                %td Disable all recipes for this item.
+                                %td
+                                    %a{:href => "/reference/inventory#material_finder"} Material Name
+
+                %h5 Recipe Attributes
+                .table-responsive
+                    %table.table.table-striped.table-condensed
+                        %thead
+                            %tr
+                                %th Attribute
+                                %th Description
+                                %th Value
+                        %tbody
+                            %tr
+                                %td
+                                    %code override
+                                %td
+                                    Override all vanilla recipes resulting in this material and data.
```

Comment:
"Disable all vanilla recipes for the same item. This is just a convenient alternative to the <disable> element."